app/presenters/rummager_presenters.rb

```diff
@@ -0,0 +1,37 @@
+module RummagerPresenters
+  # Present content for the government search index
+  def self.present_all_government_content
+    Enumerator.new do |y|
```

Comment:
Can you show me what you mean tomorrow? Don't think I've used `yielder` before...